Traceback (most recent call last):
  File "/Users/ruicheng/Documents/GitHub/WorkflowBench/ultra_parallel_runner_debug.py", line 404, in <module>
    main()
  File "/Users/ruicheng/Documents/GitHub/WorkflowBench/ultra_parallel_runner_debug.py", line 385, in main
    runner = DebugUltraParallelRunner(debug_log_dir=args.debug_log_dir)
  File "/Users/ruicheng/Documents/GitHub/WorkflowBench/ultra_parallel_runner_debug.py", line 33, in __init__
    from ultra_parallel_runner import UltraParallelRunner
  File "/Users/ruicheng/Documents/GitHub/WorkflowBench/ultra_parallel_runner.py", line 724
    model=model,
    ^^^^^^^^^^^
SyntaxError: keyword argument repeated: model
